2021-12-26 16:56:20 INFO   OctoBot Launcher     cli.py:182      Version : 0.4.3
2021-12-26 16:56:20 DEBUG  OctoBot Launcher     cli.py:102      Running on Windows:10:AMD64 with python
2021-12-26 16:56:20 INFO   OctoBot Launcher     cli.py:117      Loading config files...
2021-12-26 16:56:20 INFO   Configuration        configuration.py:99       Using DailyTrading profile.
2021-12-26 16:56:20 INFO   OctoBot Launcher     cli.py:89       Disclaimer accepted by user.
2021-12-26 16:56:21 INFO   telethon.crypto.libssl libssl.py:79       Failed to load SSL library: <class 'OSError'> (no library called "ssl" found)
2021-12-26 16:56:21 INFO   telethon.crypto.aes  aes.py:25       cryptg module not installed and libssl not found, falling back to (slower) Python encryption
2021-12-26 16:56:22 DEBUG  Commands             commands.py:101      OctoBot tentacles are up to date.
2021-12-26 16:56:22 ERROR  root                 logger.py:46         File "start.py", line 21, in <module>
    main(sys.argv[1:]) # octobot.cli 에서 main 함수
  File "C:\Users\hoon3\GitHub\TEST\octobot\cli.py", line 405, in main
    args.func(args)
  File "C:\Users\hoon3\GitHub\TEST\octobot\cli.py", line 228, in start_octobot
    bot.community_auth.clear_cache()
NoneType: None
2021-12-26 16:56:22 ERROR  root                 logger.py:47       <class 'AttributeError'>: 'OctoBot' object has no attribute 'community_auth'
NoneType: None
